src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,13 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
-        return null;
+    public static User findByEmail(String email) {
```

Comment:
It is a bed practice to use static methods 
```suggestion
    public User findByEmail(String email) {
```